decidim-core/app/models/decidim/taxonomy.rb

```diff
@@ -113,6 +113,7 @@ def set_default_weight
 
     def forbid_cycles
       return unless parent_id
+      return if parent.part_of.compact_blank.empty?
```

Comment:
Can you add a spec for this in decidim-core/spec/models/decidim/taxonomy_spec.rb?

Reply:
you can see the error in the Core pipeline error of [22e4798](https://github.com/decidim/decidim/pull/14850/commits/22e4798b236e445f7fccd651474ad898fb8dc510)

Reply:
Ah if it is related to FactoryBot and it doesn't introduces changes of behavior in the app then we can ignore that